oss-fuzz/arvo_data.py

```diff
@@ -0,0 +1,350 @@
+    'pcapplusplus': "PcapPlusPlus",
+    'skia-ftz': 'skia',
+}
+
+
+def update_resource_info(item_name: str, item_url: str,
+                         item_type: str) -> Tuple[str, str, str]:
+  """Update resource information based on configuration tables.
+    
+    Args:
+        item_name: Name of the resource item.
+        item_url: URL of the resource.
+        item_type: Type of the resource.
+        
+    Returns:
+        Tuple of (updated_name, updated_url, updated_type).
+    """
+  if item_name in CHANGED_KEY:
+    item_name = CHANGED_KEY[item_name]
+
+  if item_name in UPDATE_TABLE:
+    resource_type = CHANGED_TYPE.get(item_name, 'git')
+    return item_name, UPDATE_TABLE[item_name], resource_type
+  else:
+    return item_name, item_url, item_type
+
+
+def dockerfile_cleaner(dockerfile_path: Union[str, Path]) -> None:
+  """Clean dockerfile by removing git branch-specific arguments.
+    
+    Args:
+        dockerfile_path: Path to the Dockerfile to clean.
+    """
+  dft = DockerfileModifier(dockerfile_path)
+  dft.replace(r'(--single-branch\s+)', "")  # --single-branch
+  dft.replace(r'(--branch\s+\S+\s+|-b\s\S+\s+|--branch=\S+\s+)',
+              "")  # remove --branch or -b
+  dft.flush()
+
+
+def fix_dockerfile(dockerfile_path: Union[str, Path],
+                   project: Optional[str] = None) -> bool:
+  """Fix the dockerfile for specific projects and general issues.
+    
+    Args:
+        dockerfile_path: Path to the Dockerfile to fix.
+        project: Name of the project for project-specific fixes.
+        
+    Returns:
+        True if fixes were applied successfully, False otherwise.
+    """
+
+  def _x265_fix(dft: DockerfileModifier) -> None:
+    """Apply x265-specific fixes to the dockerfile modifier."""
+    # The order of following two lines matters
+    dft.replace(
+        r'RUN\shg\sclone\s.*bitbucket.org/multicoreware/x265\s*(x265)*',
+        "RUN git clone "
+        "https://bitbucket.org/multicoreware/x265_git.git x265\n")
+    dft.replace(
+        r'RUN\shg\sclone\s.*hg.videolan.org/x265\s*(x265)*', "RUN git clone "
+        "https://bitbucket.org/multicoreware/x265_git.git x265\n")
+
+  dockerfile_cleaner(dockerfile_path)
+  dft = DockerfileModifier(dockerfile_path)
+
+  # Some dockerfile forgets to apt update before apt install
+  # and we have to install/set ca-certificate/git sslVerify to avoid
+  # certificates issues
+  # TODO: improve regex
+  dft.replace_once(
+      r'RUN apt', "RUN apt update -y && apt install git ca-certificates -y && "
+      "git config --global http.sslVerify false && "
+      "git config --global --add safe.directory '*'\nRUN apt")
+  dft.str_replace_all(GLOBAL_STR_REPLACE)
+
+  # Apply project-specific hacks that solve building/compiling problems
+  if project == "lcms":
+    # TODO: improve this tmp patch
+    dft.replace(r'#add more seeds from the testbed dir.*\n', "")
+  elif project == 'wolfssl':
+    dft.str_replace(
```

Comment:
is it possible to structure these into something like:

```
hacks/wolfssl.py
hacks/skia.py
```

etc to do all project-specific hacks in a single file (i.e. fixing build scripts, Dockerfiles)?